### PR TITLE
Ansible - Logging Metrics

### PR DESCRIPTION
**Pull request by rambleraptor**
Logging Metrics on Ansible

This fixes two bugs: one in an incorrect self link in api.yaml and another in Ansible code.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

**Comment by rambleraptor** [comment]
@SirGitsalot - normal code review
@rileykarson - this creates a TF diff. ptal

File: products/logging/api.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ objects:
   - !ruby/object:Api::Resource
     name: "Metric"
     base_url: projects/{{project}}/metrics
-    self_link: "projects/{{project}}/metrics/{{%name}}"
```
Comment (rileykarson) [inline products/logging/api.yaml]: This is actually correct! We added `%` here to indicate "URL-encode this value", and added support for doing that in TF. Looking at blame will probably give you enough context for Ansible.

Comment (rambleraptor) [reply products/logging/api.yaml]: Sounds good. I'll make that change + document the % thing.

**Comment by rambleraptor** [comment]
No more TF diff. I no longer need anything from @rileykarson. Thanks for your help!

**Pull request merged**